src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/AADUtility.cs

```diff
@@ -17,10 +20,185 @@ public static async Task<string> AzureActiveDirectoryAuthenticationCallback(stri
             AuthenticationResult result = await authContext.AcquireTokenAsync(resource, clientCred);
             if (result == null)
             {
-                throw new InvalidOperationException($"Failed to retrieve an access token for {resource}");
+                throw new Exception($"Failed to retrieve an access token for {resource}");
```

Comment:
Why we change the exception type here?

Reply:
I didn't think "InvalidOperationException" is an appropriate exception type here.